shared/cfg/guest-os/Linux/Fedora/25.x86_64.cfg

```diff
@@ -0,0 +1,27 @@
+- 25.x86_64:
+    image_name = images/f25-64
+    vm_arch_name = x86_64
+    os_variant = fedora25
+    # Installation works fine with mem=1024 on methods such as cdrom
+    # but fails ("No space left on device") with methods such as url.
+    # anyway, let's use a safe default
+    mem = 2048
```

Comment:
On few places in `guest_os` there are `mem =` lines, but they are conditional only for `whql` variants. And no, the `subtests.cfg` is included before `guest_os`...

```
include base.cfg
include subtests.cfg
include machines.cfg
include guest-os.cfg
include guest-hw.cfg
include cdkeys.cfg
include virtio-win.cfg
```

So please do put this into the `unattended_install.url` section bellow.